### ISSUE TITLE
interpolation isn't activated when the video is started with vaapi-copy or vaapi.

### ISSUE THREAD

**Issue by laichiaheng** [mpv-player/mpv#5882]
### mpv version and platform
```
mpv 0.28.0-583-gfae9d7744c Copyright © 2000-2018 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
ffmpeg library versions:
   libavutil       56.18.102
   libavcodec      58.19.104
   libavformat     58.17.100
   libswscale      5.2.100
   libavfilter     7.24.100
   libswresample   3.2.100
ffmpeg version: N-91172-gebf85d3190
```
### Reproduction steps
open the file, and play it.
### Expected behavior
interpolation is activated when the video starts to play.
### Actual behavior
I need to adjust the timeline to make it starts to work.
### Log file
https://0x0.st/s_qu.txt

### Before I move the timeline:
![before](https://user-images.githubusercontent.com/9532105/40905410-730b24fa-6810-11e8-8a91-71b5ca511b0a.png)
### After I move the timeline:
![after](https://user-images.githubusercontent.com/9532105/40905440-84ff109a-6810-11e8-9204-32b625e6da94.png)

**Comment by laichiaheng** [comment]
If the radius of ravu-chroma-left is 4, it happens, but it doesn't happen to ravu-r3-chroma-left

**Comment by Akemi** [comment]
> [   0.954][w][cplayer] Audio/Video desynchronisation detected! Possible reasons include too slow
> [   0.954][w][cplayer] hardware, temporary CPU spikes, broken drivers, and broken files. Audio
> [   0.954][w][cplayer] position will not match to the video (see A-V status field).
> [   0.955][w][cplayer] 
> [   0.955][v][cplayer] Change display sync audio drift: 1
> [   0.989][d][vo/gpu] Resizing texture: 1920x1080
> [   0.995][v][cplayer] Video sync mode disabled.

because of this. it's too slow to start, drops a few frames at the beginning and disables video sync. seeking once resets this. since ravu-r3 is less processing intensive it's faster on startup. it would probably nice if mpv attempts to re-enable display-sync after 1 sec and that up to 3 times, or something similar.

as a workaround start the video paused and manually un-pause after everything is initialised.

**Comment by laichiaheng** [comment]
@Akemi It also doesn't happen to CPU decoding with r4, either.

**Comment by Akemi** [comment]
i would guess since initialising a hw-decoder is an additional step that slows down the startup a bit and/or it just being slower in decoding frames. 

admittedly this isn't really the most desirable behaviour.

**Comment by Akemi** [comment]
closed in favour of #6342.
